docs/examples/skanetrafiken/Readme.md

```diff
@@ -0,0 +1,161 @@
+# Skånetrafiken Deployment Configuration
+
+This is a snapshot of Skånetrafiken deployment configuration. At Skånetrafiken we deploy our OTP
+instances as microservices to a Kubernetes cluster. Some parts of the config are missing due to
+confidentiality.
+
+## Data input files
+
+### NeTEx
+
+We are using NeTEx for transport data inside Sweden. Each night we are building new file based on
+data from our SQL database. At the moment those files are not accessible through any public
+endpoint.
+
+### GTFS
+
+We are using GTFS data for traffic inside Denmark. GTFS for danish public transport can be
+downloaded [here](https://transitfeeds.com/p/rejseplanen/705?p=1).
+
+### OSM
+
+We are using two OSM files:
+
+#### Sweden
+
+We download OSM data for whole Sweden, and then we clip northern part of it to reduce file size.
+
```

Comment:
Remove all Bash. This document should be linked to Azure SIRI. The bash has nothing to do with this. A person reading this documentation wants to know how he should configure and implement SIRI end-points/Service Bus

Reply:
Removed